txtest/txTestsPmemInit.go

```diff
@@ -6,8 +6,9 @@
 package txtest
 
 import (
-	"github.com/vmware/go-pmem-transaction/pmem"
 	"os"
+
+	"github.com/vmware/go-pmem-transaction/pmem"
 )
```

Comment:
Looks like some script moved around the imports in these files. Can you revert these changes? 
Or share which script is this so I can use it too, in case this is some Go standard.

Reply:
Hi Mohit,

These were done automatically by the editor I used - Visual Studio Code. Looking around, I found the standard convention regarding organizing imports at https://github.com/golang/go/wiki/CodeReviewComments#imports

Quoting from the wiki - 

> Imports are organized in groups, with blank lines between them. The standard library packages are always in the first group.

```
package main

import (
	"fmt"
	"hash/adler32"
	"os"

	"appengine/foo"
	"appengine/user"

        "github.com/foo/bar"
	"rsc.io/goversion/version"
)
```